Bindings_Mainline.xml

```diff
@@ -1,12 +1,12 @@
 <Bindings>
-  <Binding name="BAGANATOR_OPEN_BANK" category="Baganator">
-    Baganator.CallbackRegistry:TriggerEvent("BankShow")
+  <Binding name="BAGANATOR_TOGGLE_BANK" category="Baganator">
+    Baganator.CallbackRegistry:TriggerEvent("BankToggle")
   </Binding>
-  <Binding name="BAGANATOR_OPEN_WARBAND_BANK" category="Baganator">
-    Baganator.CallbackRegistry:TriggerEvent("BankShow", 1)
+  <Binding name="BAGANATOR_TOGGLE_WARBAND_BANK" category="Baganator">
+    Baganator.CallbackRegistry:TriggerEvent("BankToggle", 1)
   </Binding>
-  <Binding name="BAGANATOR_OPEN_GUILD_BANK" category="Baganator">
-    Baganator.CallbackRegistry:TriggerEvent("GuildShow")
+  <Binding name="BAGANATOR_TOGGLE_GUILD_BANK" category="Baganator">
+    Baganator.CallbackRegistry:TriggerEvent("GuildToggle")
```

Comment:
Changing the bindings from `BAGANATOR_OPEN_BANK` to `BAGANATOR_TOGGLE_BANK` will break everyone's existing key bindings, so they'll have to re-map all key bindings when this update goes out.